modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/podcast/PodcastSettingsPage.kt

```diff
@@ -193,8 +230,102 @@ internal fun PodcastSettingsPage(
+                onChangeSkipFirst(value)
+                isSkipFirstDialogOpen = false
+            },
+            onDismiss = { isSkipFirstDialogOpen = false },
+        )
+    }
+
+    if (isSkipLastDialogOpen) {
+        ChangeSkipDurationDialog(
+            title = stringResource(LR.string.podcast_settings_skip_last),
+            placeholder = stringResource(LR.string.seconds_label),
+            initialDuration = uiState?.podcast?.skipLastSecs?.seconds ?: Duration.ZERO,
+            onConfirm = { value ->
+                onChangeSkipLast(value)
+                isSkipLastDialogOpen = false
+            },
+            onDismiss = { isSkipLastDialogOpen = false },
+        )
+    }
 }
 
+@Composable
+private fun ChangeSkipDurationDialog(
+    title: String,
+    placeholder: String,
+    initialDuration: Duration,
+    onConfirm: (String) -> Unit,
+    onDismiss: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Dialog(onDismissRequest = onDismiss) {
+        val textFieldState = rememberTextFieldState(initialText = initialDuration.inWholeSeconds.toString())
+        val focusRequester = remember { FocusRequester() }
+        LaunchedEffect(focusRequester) {
+            focusRequester.requestFocus()
+        }
+
+        Card(
+            modifier = modifier
+                .wrapContentWidth()
+                .wrapContentHeight()
+                .padding(16.dp),
+            shape = RoundedCornerShape(16.dp),
+        ) {
+            Column(
+                modifier = Modifier
+                    .wrapContentHeight()
+                    .padding(16.dp),
+                verticalArrangement = Arrangement.Center,
+                horizontalAlignment = Alignment.CenterHorizontally,
+            ) {
+                TextH30(
+                    text = title,
+                    modifier = Modifier.padding(16.dp),
+                )
+                FormField(
+                    state = textFieldState,
+                    placeholder = placeholder,
+                    keyboardOptions = keyboardOptions,
+                    onImeAction = { onConfirm(textFieldState.text.toString()) },
+                    modifier = Modifier.focusRequester(focusRequester),
+                )
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    horizontalArrangement = Arrangement.End,
+                ) {
+                    TextButton(
+                        onClick = { onDismiss() },
+                        modifier = Modifier.padding(8.dp),
+                    ) {
+                        Text(stringResource(LR.string.cancel))
+                    }
+                    TextButton(
+                        onClick = { onConfirm(textFieldState.text.toString()) },
+                        modifier = Modifier.padding(8.dp),
+                    ) {
+                        Text(stringResource(LR.string.save))
+                    }
+                }
+            }
+        }
+    }
+}
```

Comment:
Should we use a common component for entering seconds so they all look the same, and if we need to change the style later, we only have to do it in one spot? 

| Podcast settings | General settings |
| --- | --- |
| <img width="1198" height="2531" alt="Image" src="https://github.com/user-attachments/assets/da4d6624-fdb9-4136-be55-17b0caa63c50" /> | <img width="1198" height="2531" alt="Image" src="https://github.com/user-attachments/assets/8f281f05-6211-4bb8-955e-5618b756b492" /> |